rest/service/dashboardTemplate.go

```diff
@@ -128,6 +128,33 @@ var (
 				Title: "Recently visited",
 			},
 		},
+		models.Openshift: models.ModuleFederationMetadata{
```

Comment:
The build is failing in these models, I guess we need to use capital `S`?
```suggestion
		models.OpenShift: models.ModuleFederationMetadata{
```